src/library_egl.js

```diff
@@ -645,12 +643,12 @@ var LibraryEGL = {
   eglGetCurrentSurface: function(readdraw) {
     if (readdraw == 0x305A /* EGL_READ */) {
       return EGL.currentReadSurface;
-    } else if (readdraw == 0x3059 /* EGL_DRAW */) {
+    }
+    if (readdraw == 0x3059 /* EGL_DRAW */) {
```

Comment:
(and here)

Reply:
Actually, I'm happy to revert those 3 cases for now, if you are OK with the rest of the change landing?

Reply:
I don't feel strongly, I guess, if you prefer the other form.

Reply:
I reverted the three places you mentioned..